### ISSUE TITLE
Add new database APIs

### ISSUE THREAD

**Issue by alyphen** [RP-Kit/RPKit#325]
Add neater database APIs that allow  calls like:
```kotlin
database().table(RPKCharacterTable::class).get(id)
```
This should be possible after #239 as the database will be retrievable from a static context via `Bukkit` or `Sponge`.

**Comment by alyphen** [comment]
This is no longer relevant as each plugin now manages its own database connections.
